js/src/views/Signer/store.js

```diff
@@ -95,7 +95,11 @@ export default class SignerStore {
     this._api.parity
       .localTransactions()
       .then((localTransactions) => {
-        this.setLocalHashes(Object.keys(localTransactions));
+        const keys = Object
+          .keys(localTransactions)
+          .filter((key) => localTransactions[key].status !== 'invalid');
```

Comment:
Maybe keep them but display them in UI differently? Actually I will prepare a special `status` for canceled transactions. `invalid` can happen in some other cases as well and the whole `localTransactions` thing was created to never loose any of transactions that were sent from local node.